finalPrep/modZeroHero.js

```diff
@@ -1,40 +1,60 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
-
+var heroName = "Crime Stopper";
+var specialAbility = "Fire blast!";
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
+var greeting = "Greetings " + heroName;
+var catchphrase = `Don't make me use my ${specialAbility} on you!`;
 
 // Declare two variables - power AND energy - set to integers
+var power = 100;
+var energy = 100;
 
 // Declare two variables - fullPower AND fullEnergy
+var fullPower = power * 500;
+var fullEnergy = energy + 150;
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
 
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
-
+var isHuman = false;
+var identityConcealed = true;
 
 // Declare two variables - archEnemies AND sidekicks
+var archEnemies = ["Aragog", "Gru", "Squidward", "Smeagle", "Durza", "Jane"];
+var sidekicks = ["Blossom", "Cosmo", "Johnny Bravo", "Scooby", "Sirius", "Percy"];
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
 
 // Print the first sidekick to your console
-
+console.log(sidekicks[0]);
 // Print the last archEnemy to the console
-
+console.log(archEnemies[5]);
```

Comment:
@mschae16 You would want to use `archEnemies.length - 1` because the final index placement will be one less than the total array length since JavaScript uses 0-indexed arrays. Something along the lines of:

`var lastEnemy = archEnemies[archEnemies - 1]
console.log(lastEnemy)`